src/pe_source/data/helpers/redact_pii.py

```diff
@@ -0,0 +1,560 @@
+VT = [r"(?:(?<=\s)|(?<=^))\d{7}[a-zA-Z](?=$|\s)"]
+VA = [r"(?:(?<=\s)|(?<=^))[a-zA-Z]\d{2}-\d{2}-\d{4}(?=$|\s)"]
+WA = [r"(?:(?<=\s)|(?<=^))[a-zA-Z]{3}\*\*[a-zA-Z]{2}\d{3}[a-zA-Z]\d(?=$|\s)"]
+WV = [r"(?:(?<=\s)|(?<=^))[a-zA-Z]\d{6}(?=$|\s)"]
+WI = [r"(?:(?<=\s)|(?<=^))[a-zA-Z]\d{3}-\d{4}-\d{4}-\d{2}(?=$|\s)"]
+WY = [r"(?:(?<=\s)|(?<=^))\d{6}-\d{3}(?=$|\s)"]
+
+
+# Build detectors to find Drivers License ID
+
+
+class FL_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for FL drivers licenses."""
+
+    type = "FL_drivers_license"
+
+
+class FL_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify FL drivers licenses."""
+
+    name = "FL_drivers_license"
+    regex = re.compile("|".join(FL), re.IGNORECASE)
+    filth_cls = FL_DLFilth
+
+
+class HI_NE_VA_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for HI and NE drivers licenses."""
+
+    type = "HI_NE_VA_drivers_license"
+
+
+class HI_NE_VA_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify HI, NE and VA drivers licenses."""
+
+    name = "HI_NE_VA_drivers_license"
+    regex = re.compile("|".join(HI_NE_VA), re.IGNORECASE)
+    filth_cls = HI_NE_VA_DLFilth
+
+
+class IL_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for IL drivers licenses."""
+
+    type = "IL_drivers_license"
+
+
+class IL_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify IL drivers licenses."""
+
+    name = "IL_drivers_license"
+    regex = re.compile("|".join(IL), re.IGNORECASE)
+    filth_cls = IL_DLFilth
+
+
+class MN_FL_MD_MI_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for MN, FL, MD and MI drivers licenses."""
+
+    type = "MN_FL_MD_MI_drivers_license"
+
+
+class MN_FL_MD_MI_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify MN, FL, MD and MI drivers licenses."""
+
+    name = "MN_FL_MD_MI_drivers_license"
+    regex = re.compile("|".join(MN_FL_MD_MI), re.IGNORECASE)
+    filth_cls = MN_FL_MD_MI_DLFilth
+
+
+class MO_OK_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for MO and OK drivers licenses."""
+
+    type = "MO_OK_drivers_license"
+
+
+class MO_OK_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify MO and OK drivers licenses."""
+
+    name = "MO_OK_drivers_license"
+    regex = re.compile("|".join(MO_OK), re.IGNORECASE)
+    filth_cls = MO_OK_DLFilth
+
+
+class MD_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for MD drivers licenses."""
+
+    type = "MD_drivers_license"
+
+
+class MD_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify MD drivers licenses."""
+
+    name = "MD_drivers_license"
+    regex = re.compile("|".join(MD), re.IGNORECASE)
+    filth_cls = MD_DLFilth
+
+
+class CA_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for CA drivers licenses."""
+
+    type = "ca_drivers_license"
```

Comment:
Is there a reason why this isn't
```suggestion
    type = "CA_drivers_license"
```
to match the others above it?

Similar changes should be made below.

Reply:
@dav3r the rest of these issues are addressed in the following commit: [a0bde91](https://github.com/cisagov/pe-reports/pull/190/commits/a0bde91cdf6860f0014d3ae8d869bafba112d414)